pages/common/ghc.md

```diff
@@ -0,0 +1,23 @@
+# ghc
+
+> The Glassgow Haskell Compiler, compiles and links Haskell source files.
+
+- Find and compile all modules in the current directory:
+
+`ghc Main`
+
+- Compile a single file:
+
+`ghc {{file.hs}}`
+
+- Compile using optimization, for faster code:
+
+`ghc -O {{file.hs}}`
+
+- Stop after generating object files:
```

Comment:
Stop what? It would be nice to be a tad more descriptive here, for total beginners.